backend/src/main/java/kr/momo/domain/schedule/recommend/CandidateSchedule.java

```diff
@@ -32,14 +35,26 @@ public static List<CandidateSchedule> mergeContinuous(
                     .takeWhile(i -> i == headIdx || isSequential(i, sortedSchedules, isContinuous))
                     .map(sortedSchedules::get)
                     .toList();
-            subList.stream()
-                    .reduce(CandidateSchedule::merge)
-                    .ifPresent(mergedSchedules::add);
+            addWhenLongerOrEqualThanMinTime(subList, mergedSchedules, minSize);
             idx += subList.size();
         }
         return mergedSchedules;
     }
 
+    private static void addWhenLongerOrEqualThanMinTime(
+            List<CandidateSchedule> subList, List<CandidateSchedule> mergedSchedules, int minSize
```

Comment:
```suggestion
    private static void addIfLongerThanOrEqualToMinTime(
```
일반적으로 사용되는 메서드 명명법을 따라 볼까요? 🙂

Reply:
오호 이 부분은 흥미롭네요.
일반적으로 사용되는 메서드 명명법을 참고할 만한 곳이 있으면 공유 요청 드려도 될까요?
추후 메서드  명명에 도움이 될 것 같습니다!

Reply:
음.. 별도로 레퍼런스가 있다기보다는 보통 영문에서 `lt`, `lte`, `gt`, `gte` 와 같은 표기를 많이 사용해서 그게 자연스럽다고 느꼈던 것 같아요.

자바 계열에서는 JUnit의 검증 메서드 명명법을 참고하기는 합니다. 저 메서드 보자마자 JUnit의 `greaterThanOrEqualTo()` 가 떠올랐어요ㅋㅋㅋ